pages/dkp/kommander/2.3/projects/applications/catalog-applications/custom-applications/application-metadata/index.md

```diff
@@ -10,14 +10,15 @@ You can define how custom applications display in the DKP UI by defining a `meta
 
 You can define the following fields:
 
-| Field       | Default              | Description                                                                                     |
-| ----------- | -------------------- | ----------------------------------------------------------------------------------------------- |
-| displayName | falls back to App ID | Display name of the application for the UI.                                                     |
-| description | ""                   | Short description, should be a sentence or two, displayed in the UI on the application card.    |
-| category    | general              | 1 or more categories for this application. Categories are used to group applications in the UI. |
-| overview    |                      | Markdown overview used on the application detail page in the UI.                                |
-| icon        |                      | Base64 encoded icon SVG file used for application logos in the UI.                              |
-| scope       | workspace            | List of scopes, can be workspace and/or project currently.                                                                                               |
+| Field                  | Default              | Description                                                                                     |
+| ---------------------- | -------------------- | ----------------------------------------------------------------------------------------------- |
+| displayName            | falls back to App ID | Display name of the application for the UI.                                                     |
+| description            | ""                   | Short description, should be a sentence or two, displayed in the UI on the application card.    |
+| category               | general              | 1 or more categories for this application. Categories are used to group applications in the UI. |
+| overview               |                      | Markdown overview used on the application detail page in the UI.                                |
+| icon                   |                      | Base64 encoded icon SVG file used for application logos in the UI.                              |
+| scope                  | workspace            | List of scopes, can be workspace and/or project currently.                                      |
```

Comment:
We could say: 

> List of scopes, currently can be workspace, project, or both.

Reply:
I've changed it, thank you for that suggestion 🙌